install.sh

```diff
@@ -2201,14 +1743,12 @@ MARKETPLACE_DRIVER_REGISTRY_SCRIPTS="src/market_mad/remotes/docker_registry/impo
 ONEDB_FILES="src/onedb/fsck.rb \
             src/onedb/onedb.rb \
             src/onedb/onedb_backend.rb \
-            src/onedb/vcenter_one54.rb \
             src/onedb/sqlite2mysql.rb \
             src/onedb/database_schema.rb \
             src/onedb/fsck \
             src/onedb/onedb_live.rb"
 
-ONEDB_PATCH_FILES="src/onedb/patches/4.14_monitoring.rb \
-                   src/onedb/patches/history_times.rb"
+ONEDB_PATCH_FILES="src/onedb/patches/history_times.rb"
 
 ONEDB_SHARED_MIGRATOR_FILES="src/onedb/shared/2.0_to_2.9.80.rb \
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

ONEDB_SHARED_MIGRATOR_FILES appears unused. Verify use (or export if used externally).